templates/admin_edit_updated_template.html

```diff
@@ -36,8 +36,8 @@ <h3><span class="label label-primary">{{ template_to_update.name }}</span></h3>
                     <div class="col-xs-12">
                         <form action="" method="post">{% csrf_token %}
                             {{ form.as_p }}
-                            <a href="{% url 'admin_updated_template' template_to_update.pk %}" class="btn btn-default btn-lg" role="button">Cancel</a>
-                            <input class="btn-primary pull-right btn-lg" type="submit" value="Update" />
+                            <a href="{% url 'admin_updated_template' template_to_update.pk %}" class="btn btn-default btn-lg" role="button" id="custom-btn-default">Cancel</a>
+                            <input class="btn-primary pull-right btn-lg" type="submit" value="Update" id="custom-btn" />
```

Comment:
The `id` attribute is meant to provide a unique id to an html element. And I think they should be used so you can quickly query the DOM for that specific element. If its just being used to identify an element for styling, then it could just as well be done by adding a class. I think that may be more idiomatic.

Reply:
Makes sense, will look into it!